### PR TITLE
Bump the lowest Debian version to Bookworm in CI

### PR DESCRIPTION
**Pull request by schopin-pro**
The latest Debian stable is Bookworm, not Bullseye anymore.

### CODE REVIEW

**Comment by schopin-pro** [comment]
I greatly reduced the scope of this PR as it turns out we can't have runners that run interim Ubuntu releases, only LTS, which means our installed tests can't be run on Lunar. That means more work on #193 to fallback on old style when appropriate.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ Please see https://wiki.ubuntu.com/Apport for more details and further links.
 The files in [doc/](./doc/) document particular details such as package hooks,
 crash database configuration, or the internal data format.
 
+Platform requirements
+----------------
```
Comment (bdrung) [inline README.md:28]:
```suggestion
---------------------
```

File: README.md [hunk 2]
```diff
@@ -24,6 +24,15 @@ Please see https://wiki.ubuntu.com/Apport for more details and further links.
...
+
+The absolute minimum requirement for your distribution are Python 3.10 or
```
Comment (bdrung) [inline README.md:30]: question: Is that correct English? "The requirement are"

File: apport/packaging.py [hunk 3]
```diff
@@ -12,50 +12,7 @@
...
-            except OSError as error:
-                errno = error.errno
-        else:
-            raise OSError(errno, f"Unable to read files {', '.join(os_release_files)}")
-
-        return ret
-
-    def freedesktop_os_release() -> typing.Dict[str, str]:
-        return _parse_os_release("/etc/os-release", "/usr/lib/os-release")
+from platform import freedesktop_os_release
```
Comment (bdrung) [inline apport/packaging.py:15]: suggestion: For consistency we could `import platform` now and use `platform.freedesktop_os_release` later on.

**Pull request merged**